app/components/advanced_search/condition.html.erb

```diff
@@ -25,14 +25,21 @@
                                ),
                              ),
                              { include_blank: t("advanced_search_component.field") },
-                             { "aria-label": t("advanced_search_component.field") } %>
+                             {
+                               "aria-label": t("advanced_search_component.field"),
+                               "aria-invalid": invalid_field,
+                               "aria-describedby":
+                                 conditions_form.field_id(:field, "error"),
+                               autofocus: invalid_field,
+                             } %>
       <%= render "shared/form/field_errors",
-      errors: @condition.errors.full_messages_for(:field) %>
+      errors: @condition.errors.full_messages_for(:field),
+      id: conditions_form.field_id(:field, "error") %>
     </div>
     <% invalid_operator = @condition.errors.include?(:operator) %>
     <div
       class="
-        form-field w-1/6 @max-xl:w-auto <%= 'invalid' if invalid_operator %>
+        form-field w-1/4 @max-xl:w-auto <%= 'invalid' if invalid_operator %>
```

Comment:
Can you change `@max-xl:w-auto` to `@max-xl:w-full`, here and on line 14, and on line 8 in `value.html.erb` and then add `w-full` to line 15, the field will expand to max width on a smaller screen and look better and keep its alignment on a wider screen.

<img width="1272" height="1684" alt="Image" src="https://github.com/user-attachments/assets/b4cff266-2131-4010-a0b9-51067c0f4f7a" />